### PR TITLE
Add readme definition

### PR DESCRIPTION
**Pull request by sstevens2**
## Author: 

- Sarah Stevens

## Language: 

- English

## Terms defined:

- README

## Editor

Assign the PR based on the language to the following person:

@zkamvar  

Not sure this a great definition but thought it might be a good start to getting `README` added.  We'd like to link to the definition in the [Jekyll Pages Lesson Glossary](https://carpentries-incubator.github.io/building-websites-with-jekyll-and-github-or-gitlab/reference.html)

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -4575,6 +4575,15 @@
...
       change. Reactive variables are used extensively in [Shiny](#shiny).
 
+- slug: readme
+  ref:
+    - markdown
+    - vignette
+  en:
+    term: "README"
+    def: >
+      A file which contains the important information about a project or software package.
```
Comment (zkamvar) [inline glossary.yml:4585]: This matches the style of other definitions (e.g. Makefile) better.

```suggestion
      A plain text file containing important information about a project or software package.
```

**Review by zkamvar — COMMENTED** [review]
LGTM. I only have the one suggestion about phrasing.

**Comment by sstevens2** [comment]
That change looked great!  I committed it.

**Review by zkamvar — APPROVED** [review]

**Comment by zkamvar** [comment]
Excellent! Now we just need to wait for Github Actions to sort itself out and I'll merge it :)

Thank you for the contribution!

**Pull request merged**